examples/html-kitchen-sink/stories/react/addon-a11y.stories.js

```diff
@@ -0,0 +1,101 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/html';
+
+import { Form } from '@storybook/components';
+import BaseButton from './components/BaseButton';
+import DelayedRender from './components/DelayedRender';
+import Button from './components/addon-a11y/Button';
+
+const text = 'Testing the a11y addon';
+const image = 'http://placehold.it/350x150';
+// eslint-disable-next-line no-script-url
+const href = 'javascript:void 0';
+
+storiesOf('React|A11y/BaseButton', module)
+  .addParameters({
+    framework: { react: true },
```

Comment:
`framework: 'react'` doesn't work? seems like it would be worth modifying the parameters code if it doesn't.

Reply:
@tmeasday do you remember why parameters have to be objects?

Reply:
It was just so we can do inheritance btw global/kind/story parameters via composing the internal parts. I think we could make parameters scalar, with of course the understanding that setting them in a more specific place would override the less specific completely.